lib/openxml/drawingml/properties/blip.rb

```diff
@@ -0,0 +1,28 @@
+module OpenXml
+  module DrawingML
+    module Properties
+      class Blip < OpenXml::Properties::ComplexProperty
+        include HasChildren
+        include ContainsProperties
+        namespace :a
+        tag :blip
+
+        attribute :compression_state, one_of: %i{ email hqprint none print screen }, displays_as: :cstate
+
+        with_namespace :r do
+          attribute :embed, expects: :string
+          attribute :link, expects: :string
+        end
+
+        # TODO: All the various blip child elements as properties
+
+        property :extension_list
+
+        def render?
+          true
+        end
```

Comment:
❔ What does this do?

Reply:
By default, a property is only rendered if it a) has attributes defined or b) has children of some form. There are a handful of exceptions (where you'd want a blank tag to render anyways), and this is one of them. It forces the render by overriding the `render?` method.

Reply:
Ah! Ok. Thanks!